src/test/regress/expected/subquery_and_cte.out

```diff
@@ -470,7 +470,7 @@ FOR i IN 1..3 LOOP
 			GROUP BY
 				user_id, cte.value_2);
 	EXCEPTION WHEN OTHERS THEN
-		IF SQLERRM LIKE 'failed to execute task%' THEN
+		IF SQLERRM LIKE 'more than one row returned by a subquery%%' THEN
```

Comment:
mainly just wondering: do we return a useful error code here?

Reply:
Yes, we return the same error code that Postgres does:
```
ERROR:  21000: more than one row returned by a subquery used as an expression
```

which is 

`21000 | cardinality_violation`